contracts/JBController.sol

```diff
@@ -100,92 +94,92 @@ contract JBController3_1 is
     // ------------------------- external views -------------------------- //
     //*********************************************************************//
 
-    /// @notice Gets the current total amount of outstanding tokens for a project.
-    /// @param _projectId The ID of the project to get total outstanding tokens of.
-    /// @return The current total amount of outstanding tokens for the project.
-    function totalOutstandingTokensOf(uint256 _projectId)
+    /// @notice Gets the current token supply of a project, including pending reserved tokens.
+    /// @param _projectId The ID of the project to get the total token supply of.
+    /// @return The current total token supply of the project, including pending reserved tokens that have not been sent to splits yet.
+    function totalTokenSupplyWithReservedTokensOf(uint256 _projectId)
         external
         view
         override
         returns (uint256)
     {
         // Add the reserved tokens to the total supply.
-        return tokenStore.totalSupplyOf(_projectId) + reservedTokenBalanceOf[_projectId];
+        return tokens.totalSupplyOf(_projectId) + pendingReservedTokenBalanceOf[_projectId];
```

Comment:
Should this be renamed? Appears it's not the total supply.

Reply:
meh. as far as JBTokens is concerned it is the total supply. just needs to add the pending reserved.

Reply:
Keeping totalSupplyOf for now. But I think this is fair and worth revisiting if we can think of something really clear.